groups/tests/__init__.py

```diff
@@ -0,0 +1,9 @@
+import pytest
+
+from MangAdventure.tests import MangadvTestBase
+
+
+@pytest.mark.usefixtures("django_db_setup")
+class GroupsTestBase(MangadvTestBase):
+    def setup_method(self):
+        super().setup_method()
```

Comment:
Yeah cause otherwise anything done in the base test class's setup_method (`MangadvTestBase`) doesn't propagate forward (and __init__ is not an option for pytest test classes.)